### ISSUE TITLE
Aktualizace z GitHubu

### ISSUE THREAD

**Issue by kalich5** [skaut/dsw-oddil#133]
Narazil jsem na chybu při aktualizaci. Když je nainstalovaná starší šablona a aktualizuje se na novější. Nejsou odstraněné soubory, které v té nové už nejsou. Zůstanou tam a šablona nefunguje korektně.

@vojta001 můžeš se na to podívat. Děkuju

**Comment by vojta001** [comment]
Jsi si tím jistý? Zatím jsem to nezkoušel, ale např. [tento článek](https://halfelf.org/2011/how-the-wordpress-upgrade-works) tvrdí opak.

> Most of the time, we all see the plugin and theme installer, where it downloads the plugin to /wp-content/upgrade/, extracts the plugin, deletes the old one, and copies up the new one.

**Comment by kalich5** [comment]
Jo, když jsem nainstaloval první verzi, která to uměla, byly tam widgety v hlavičce. Ty jsme pak odstranily a čisté poslední verzi nejsou. Když udělám upgrade, tak tam jsou. A i další soubory, které v té aktualizace zmizí.
